tests/data_context/test_data_context_profilers.py

```diff
@@ -3,25 +3,13 @@
 import pytest
 from marshmallow import ValidationError
 
-import great_expectations.exceptions as ge_exceptions
 from great_expectations.core.batch import BatchRequest
 from great_expectations.data_context import DataContext
 from great_expectations.data_context.store.profiler_store import ProfilerStore
 from great_expectations.rule_based_profiler.config import RuleBasedProfilerConfig
 from great_expectations.rule_based_profiler.rule_based_profiler import RuleBasedProfiler
 
 
-def test_list_profilers_raises_configuration_error(empty_data_context: DataContext):
-    with mock.patch(
-        "great_expectations.data_context.DataContext.profiler_store",
-    ) as mock_profiler_store:
-        mock_profiler_store.__get__ = mock.Mock(return_value=None)
-        with pytest.raises(ge_exceptions.StoreConfigurationError) as e:
-            empty_data_context.list_profilers()
-
-    assert "not a configured store" in str(e.value)
-
-
```

Comment:
Deleted this test as it was duplicatively testing the `self.profiler_store` property